src/test/java/uk/gov/digital/ho/hocs/audit/entrypoint/BaseExportResourceTest.java

```diff
@@ -34,7 +34,7 @@
 import static org.springframework.test.context.jdbc.SqlConfig.TransactionMode.ISOLATED;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ActiveProfiles("local")
+@ActiveProfiles("timeline")
```

Comment:
```suggestion
@ActiveProfiles({"local", "timeline"})
```
If local isn't needed here anymore that should be a separate change.